c_cxx/README.md

```diff
@@ -10,59 +10,69 @@ This directory contains a few C/C++ sample applications for demoing onnxruntime
 
-```
-build.bat --config RelWithDebInfo --build_dir .\build  --build_shared_lib --parallel  --cmake_extra_defines CMAKE_INSTALL_PREFIX=c:\dev\ort_install
+```bat
+.\build.bat --config RelWithDebInfo --build_shared_lib --parallel
 ```
 
-By default products of the build on Windows go to .\build\Windows\<config> folder. In the case above it would be .\build\RelWithDebInfo since the build folder is mentioned explicitly.
-If you did not specify alternative installation location above you would need to open an elevated command prompt to install onnxruntime.
-Run the following commands.
+By default the build output will go in the `.\build\Windows\<config>` folder, and
+"C:\Program Files\onnxruntime" will be the installation location.
 
+You can override the installation location by specifying CMAKE_INSTALL_PREFIX via the cmake_extra_defines parameter.
+
+e.g.
+```bat
+.\build.bat --config RelWithDebInfo --build_shared_lib --parallel --cmake_extra_defines CMAKE_INSTALL_PREFIX=D:\onnxruntime
 ```
-cmake --install .\build\RelWithDebInfo --config RelWithDebInfo
+
+Run the below command to install onnxruntime.
+If installing to "C:\Program Files\onnxruntime" you will need to run the command from an elevated command prompt.
 
+```
+cmake --install .\build\Windows\RelWithDebInfo --config RelWithDebInfo
 ```
 
 ## Build the samples
 
-Open Developer Command Prompt for Visual Studio version you are going to use, change your current directory to samples\c_cxx, then run
+Open Developer Command Prompt or Developer PowerShell for the Visual Studio version you are going to use, 
+change your current directory to samples\c_cxx, and run the below command. 
+
+- You may omit the "-DLIBPNG_ROOTDIR=..." argument if you don't have the libpng library.
+- You may omit "-DONNXRUNTIME_ROOTDIR=..." if you installed to "C:\Program Files\onnxruntime", 
+otherwise adjust the value to match your ONNX Runtime install location.
+- You may append "-Donnxruntime_USE_CUDA=ON" or "-Donnxruntime_USE_DML=ON" to the last command args if your 
+onnxruntime binary was built with CUDA or DirectML support respectively.
+
 ```bat
 mkdir build && cd build
-cmake .. -A x64 -T host=x64 -DLIBPNG_ROOTDIR=C:\path\to\your\libpng\binary -DONNXRUNTIME_ROOTDIR=c:\dev\ort_install
+cmake .. -A x64 -T host=x64 -DONNXRUNTIME_ROOTDIR=D:\onnxruntime -DLIBPNG_ROOTDIR=C:\path\to\your\libpng\binary
 ```
-You may omit the "-DLIBPNG_ROOTDIR=..." argument if you don't have the libpng library.
-You may omit "-DONNXRUNTIME_ROOTDIR=..." if you installed to a default location.
 
-You may append "-Donnxruntime_USE_CUDA=ON" or "-Donnxruntime_USE_DML=ON" to the last command args if your onnxruntime binary was built with CUDA or DirectML support respectively.
+You can open and build the solution using Visual Studio,
 
-You can then either open the solution in a Visual Studio and build it from there
 ```bat
 devenv onnxruntime_samples.sln
 ```
-Or build it using msbuild
+
+or build it from the command line using msbuild
```

Comment:
using `cmake --build` could be more cross platform. but this readme seems pretty windows-specific now.

Reply:
Yeah. All the info here is Windows specific. At some point it would be good to fill in similar info for other platforms, but the goal of this PR is just to fix the existing setup that's broken.